src/components/general/FeedDisplay/FeedSettingsModal.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import styles from './FeedDisplay.module.css';
+
+export default function FeedSettingsModal({ onClose, feed }) {
+  const [feedData, setFeedData] = useState({ alias: '', url: '', pollTimer: '' });
+
+  useEffect(() => {
+    if (feed) {
+      setFeedData({
+        alias: feed.alias || '',  // Default to empty string if undefined
+        url: feed.url || '',      // Default to empty string if undefined
+        pollTimer: feed.poll_interval || '' // Default to empty string if undefined
+      });
+    }
+  }, [feed]);
+
+  const handleChange = (e) => {
+    setFeedData({
+      ...feedData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleUpdate = () => {
+    // Handle update logic here
```

Comment:
I assume this will invoke an "update" db function (yet to be written)? Ditto for handleDelete?

Reply:
Oh, I know I said _maybe_ on discord earrlier but 100% yes now, this and a few other spots I "dummy coded" placeholders so that it is easy to adjust once the DB layout is a bit more finalized.

This and the other spots will handle user updates to the DB (be it deletes, adds, or updated)